tidy3d/components/simulation.py

```diff
@@ -1330,19 +1332,38 @@ def plot(
             Opacity of the monitors. If ``None``, uses Tidy3d default.
         ax : matplotlib.axes._subplots.Axes = None
             Matplotlib axes to plot on, if not specified, one is created.
+        hlim : Tuple[float, float] = None
+            The x range if plotting on y or z planes, y range if plotting on x plane.
+        vlim : Tuple[float, float] = None
+            The z range if plotting on x or y planes, y plane if plotting on z plane.
 
         Returns
         -------
         matplotlib.axes._subplots.Axes
             The supplied or created matplotlib axes.
         """
-
-        ax = self.plot_structures(ax=ax, x=x, y=y, z=z)
-        ax = self.plot_sources(ax=ax, x=x, y=y, z=z, alpha=source_alpha)
-        ax = self.plot_monitors(ax=ax, x=x, y=y, z=z, alpha=monitor_alpha)
-        ax = self.plot_symmetries(ax=ax, x=x, y=y, z=z)
-        ax = self.plot_pml(ax=ax, x=x, y=y, z=z)
-        ax = self._set_plot_bounds(ax=ax, x=x, y=y, z=z)
+        # if no hlim and/or vlim given, the bounds will then be the usual pml bounds
+        axis, _ = self.parse_xyz_kwargs(x=x, y=y, z=z)
+        _, (hmin, vmin) = self.pop_axis(self.bounds_pml[0], axis=axis)
+        _, (hmax, vmax) = self.pop_axis(self.bounds_pml[1], axis=axis)
+
+        # account for unordered limits
+        if hlim is None:
+            hlim = (hmin, hmax)
+        if vlim is None:
+            vlim = (vmin, vmax)
+
+        if hlim[0] > hlim[1]:
+            raise Tidy3dError("Error: hmin > hmax")
```

Comment:
in error messages (things that get printed more generally) we put regular single quotes `'` around variables, so 
```
Error: 'hmin' > 'hmax'
```
There's a discussion [here](https://www.notion.so/flexcompute/Python-Style-Guide-afac5dbbf949463e89c18fe39b7453c9) if you want to read about it.